### ISSUE TITLE
Search Results Not Red Or Focused in NVDA 2019.2

### ISSUE THREAD

**Issue by ntcarver0** [nvaccess/nvda#10109]
### Steps to reproduce:
1. Open the Windows start menu.
2. Search for something.

### Actual behavior:
NVDA plays the "suggestions opened" sound, owever no suggestions are vocalized or focused.

### Expected behavior:
NVDA should play the "suggestions opened" sound, and should also focus and vocalize search results automatically.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2019.2

#### Windows version:
Version	10.0.18362 Build 18362

#### Name and version of other software in use when reproducing the issue:
Windows 10 App Essentials (wintenApps) Version 19.08

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA behaves as expected in all other versions that have this feature until 2019.2.

[20190816-noah-carver-nvda.log](https://github.com/nvaccess/nvda/files/3511151/20190816-noah-carver-nvda.log)

**Comment by Adriani90** [comment]
I cannot reproduce this issue in Windows 10 1903 with windows 10 20190816dev.
cc: @josephsl

**Comment by ntcarver0** [comment]
> 
> 
> I cannot reproduce this issue in NVDA 2019.2, Windows 10 1903 and windows 10 app essentials 20190816dev.
> cc: @josephsl

A system restart didn't solve the issue, but when I killed explorer.exe then ran it again the issue went away. Everything works as expected now.

**Comment by Adriani90** [comment]
Thanks for the swift update. I will close this issue as works for me but if you still encounter this issue, feel free to comment on it and we can reopen it. Thanks.
